pkgs/kubernetes-sigs/controller-tools/controller-gen/registry.yaml

```diff
@@ -0,0 +1,11 @@
+packages:
+  - type: go_install
+    name: kubernetes-sigs/controller-tools/controller-gen
+    aliases:
+      - name: kubernetes-sigs/kubebuilder/controller-gen
+    path: sigs.k8s.io/controller-tools/cmd/controller-gen
+    repo_owner: kubernetes-sigs
+    repo_name: controller-tools
+    description: controller-gen - Generate Kubernetes API extension resources and code.
```

Comment:
https://github.com/aquaproj/aqua-registry/blob/main/CONTRIBUTING.md#remove-a-period-from-the-end-of-the-description

```suggestion
    description: controller-gen - Generate Kubernetes API extension resources and code
```

Reply:
Ah, sorry!  Should I update?

Reply:
Yes. Please update.
You have to update `registry.yaml` too.